src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -18,7 +18,9 @@
     -->
 
     <ValidPinvokeMappings>$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
+    <ValidPinvokeMappings Condition="'$(UseWin32Apis)'=='true'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_Win32Apis.txt</ValidPinvokeMappings>
```

Comment:
This will be set by the project that uses the buildtools. For example, in dotnet/corefx, this will be set in https://github.com/dotnet/corefx/blob/master/dir.props by an upcoming PR.

Eventually, if most or all projects start using this option, we may consider making it the default. However to avoid a breaking change, for now projects need to opt in.